SampleScripts/PowerManagement-WorkloadDomain.ps1

```diff
@@ -254,6 +254,42 @@ Try {
         [Array]$vcfvms = @()
         [Array]$vcfvms += ($vcServer.fqdn).Split(".")[0]
 
+        # Gather VxRail Manager Details for the VI Workload Domain, if exists
+        if ($PsBoundParameters.ContainsKey("shutdown")) {
+            $vxRailCred = (Get-VCFCredential | Where-Object { $_.resource.resourceType -eq "VXRAIL_MANAGER" -and $_.resource.domainName -eq ($workloadDomain.name) -and $_.username -eq "root" })
+            if ($vxRailCred -ne $null)
+            {
+			    # Connecting to VC to get VxRail VM name
+			    if ($DefaultVIServers) {
+                        Disconnect-VIServer -Server * -Force -Confirm:$false -WarningAction SilentlyContinue | Out-Null
+                }
+                if ((Test-NetConnection -ComputerName $vcServer.fqdn -Port 443 ).TcpTestSucceeded) {
+                    Write-PowerManagementLogMessage -Type INFO -Message "Connecting to '$($vcServer.fqdn)' ..."
+                    Connect-VIServer -Server $vcServer.fqdn -Protocol https -User $vcUser -Password $vcPass -ErrorVariable $vcConnectError | Out-Null
+                    if ($DefaultVIServer.Name -eq $vcServer.fqdn) {
+					    $vxrailVMObject = Get-VM | where-object {$_.Guest.Hostname -match $vxRailCred.resource.resourceName -Or $_.Guest.Hostname -match ($vxRailCred.resource.resourceName.Split("."))[0]}
```

Comment:
```suggestion
					    $vxrailVMObject = Get-VM | Where-Bbject {$_.Guest.Hostname -match $vxRailCred.resource.resourceName -Or $_.Guest.Hostname -Match ($vxRailCred.resource.resourceName.Split("."))[0]}
```

Reply:
addressed in commit f8c5a69ef84a1a40675a91879ad0a67701fba108